`@remotion/install-whisper-cpp`: Whisper 1.7.4 will fail to install on Mac if `cmake` is not present

<!--- Provide a general summary of the issue in the Title above -->

# Bug Report 🐛

Whisper.cpp 1.7.4 fails to install when using  `@remotion/install-whisper-cpp` on Mac OS.

Steps to reproduce:

1. Make sure you don't have `cmake` installed on your system
2. Run the script

```ts
import path from "path";
import {
  downloadWhisperModel,
  installWhisperCpp,
} from "@remotion/install-whisper-cpp";

const WHISPER_VERSION = "1.7.4";
const WHISPER_MODEL = "medium";
const WHISPER_PATH = path.join(process.cwd(), "whisper.cpp");

await installWhisperCpp({
  to: WHISPER_PATH,
  version: WHISPER_VERSION,
});

await downloadWhisperModel({
  model: WHISPER_MODEL,
  folder: WHISPER_PATH,
});
```

This will error if you don't have `cmake` installed on your system:
```
HEAD is now at 8a9ad78 release : v1.7.4
cmake -B build
make: cmake: No such file or directory
make: *** [build] Error 1
46 |             cwd: cwd !== null && cwd !== void 0 ? cwd : undefined,
47 |             shell: shell !== null && shell !== void 0 ? shell : undefined,
48 |         });
49 |         child.on('exit', (code, exitSignal) => {
50 |             if (code !== 0) {
51 |                 reject(new Error(`Error while executing ${bin} ${args.join(' ')}. Exit code: ${code}, signal: ${exitSignal}`));
                            ^
error: Error while executing make . Exit code: 2, signal: null
      at <anonymous> (/Users/igor/Projects/remotion-main/packages/install-whisper-cpp/dist/install-whisper-cpp.js:51:24)
      at emit (node:events:92:22)
      at #handleOnExit (node:child_process:518:14)

Bun v1.2.6 (macOS arm64)
```

## Workaround

1. Install [Homebrew](https://brew.sh/)
2. Run `brew install cmake`

--- a/packages/docs/docs/install-whisper-cpp/index.mdx
+++ b/packages/docs/docs/install-whisper-cpp/index.mdx
@@ -67,3 +67,7 @@ for (const line of captions) {
 ## License
 
 MIT
+
+## See also
+
+- [`@remotion/openai-whisper`](/docs/openai-whisper)

--- a/packages/docs/docs/install-whisper-cpp/install-whisper-cpp.mdx
+++ b/packages/docs/docs/install-whisper-cpp/install-whisper-cpp.mdx
@@ -19,15 +19,18 @@ const {alreadyExisted} = await installWhisperCpp({
 });
 ```
 
-On Windows, if the provided version is not in semantic version format (e.g., `1.5.4`), the installation will throw an error. The function expects a precompiled semantic versioned release available for download.
+## Which Whisper.cpp version to select?
 
-On other platforms, the source is cloned from GitHub and built.
+- Generally, the source is cloned from Whisper.cpp GitHub and built.
+- On Windows, a binary is downloaded. Only semantic version format (e.g. `1.5.4` is supported).
+- On Windows, there are no binaries newer than `1.6.0` available.
+- From `1.7.3` and later, `cmake` is required for Whisper.cpp to be built.
 
-If you install Whisper.cpp into a folder which is in a Git repository, you should add the folder to the `.gitignore` file.
+## Note this
 
-If the output folder already exists, the function will not do anything and return `{ alreadyExisted: true }`.
-
-You also need to download a model for Whisper.cpp to work. You can do this with the [`downloadWhisperModel()`](/docs/install-whisper-cpp/download-whisper-model) function.
+- You should add the folder you install Whisper.cpp into to the `.gitignore` file.
+- If the output folder already exists, the function will not do anything and return `{ alreadyExisted: true }`.
+- You also need to download a model for Whisper.cpp to work. You can do this with the [`downloadWhisperModel()`](/docs/install-whisper-cpp/download-whisper-model) function.
 
 ## Options
 
@@ -64,4 +67,5 @@ Whether the folder already existed. If it did, and contains the necessary execut
 
 - [Source code for this function](https://github.com/remotion-dev/remotion/blob/main/packages/install-whisper-cpp/src/install-whisper-cpp.ts)
 - [`@remotion/install-whisper-cpp`](/docs/install-whisper-cpp)
+- [`@remotion/openai-whisper`](/docs/openai-whisper)
 - [`downloadWhisperModel()`](/docs/install-whisper-cpp/download-whisper-model)

